docs/dasharo-menu-docs/dasharo-system-features.md

```diff
@@ -145,6 +145,16 @@ Currently available options:
     + `Suspend to RAM (S3)` - standard sleep mode that suspends the machine
       state to RAM, supported on most operating systems (except Windows on 11th
       generation Intel Core processors or newer).
+- `Battery Start Treshold` - the value set by this option represents the
+percentage of battery power that the device will never fall below - it will
```

Comment:
This is not correct. The battery start charge threshold is the level above which the battery will not start charging. So if you set the start threshold to 75% and stop threshold to 80%, the laptop will start charging when the battery is below 75%, and will not start charging otherwise.

The battery can still be discharged to 0%.

Reply:
fixed in fcbafbef361ed26c88ee5bd2abcff20d4a00e782